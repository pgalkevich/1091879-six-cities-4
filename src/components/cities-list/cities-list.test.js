import React from 'react';
import renderer from 'react-test-renderer';
import CitiesList from "./cities-list";

const cities = [
  `Paris`,
  `Cologne`,
  `Brussels`,
  `Amsterdam`,
  `Hamburg`,
  `Dusseldorf`
];

it(`CitiesList is rendered correctly`, () => {
  const tree = renderer.create(
      <CitiesList cities={cities} currentCity={`Amsterdam`} handler={() => {}} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
