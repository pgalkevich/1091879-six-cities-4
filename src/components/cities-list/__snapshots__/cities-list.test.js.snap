// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList is rendered correctly 1`] = `
Array [
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Hamburg
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Paris
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Dusseldorf
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Brussels
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Cologne
      </span>
    </a>
  </li>,
]
`;
